lib/Runtime/Library/JavascriptSet.cpp

```diff
@@ -88,12 +88,9 @@ namespace Js
 
         if (iter != nullptr)
         {
-            Var nextItem;
-
-            while (JavascriptOperators::IteratorStepAndValue(iter, scriptContext, &nextItem))
```

Comment:
Is `IteratorStepAndValue` used anywhere anymore?  Can we delete it and rename `DoIteratorStepAndValue` to `IteratorStepAndValue`?